bundle/compliance/cis_aws/rules/cis_1_8/data.yaml

```diff
@@ -0,0 +1,67 @@
+metadata:
+  id: 0674190c-677c-5f17-bcc8-f60e913eb9d6
+  name: Ensure IAM password policy requires minimum length of 14 or greater
+  profile_applicability: "* Level 1"
+  description: Password policies are, in part, used to enforce password complexity
+    requirements. IAM password policies can be used to ensure password are at
+    least a given length. It is recommended that the password policy require a
+    minimum password length 14.
+  version: "1.0"
+  rationale: Setting a password complexity policy increases account resiliency
+    against brute force login attempts.
+  audit: >-
```

Comment:
it seems like we use `>`  instead of `>-` in most places (`>-` wont appends a linebreak at the end)